### PR TITLE
Add AzureML devops pipeline test badges to base readme, update AML info in tests/readme

### PR DESCRIPTION
**Pull request by bethz**
### Description
- Added AzureML devops pipeline test badges to base readme
- updated AML info in tests/readme
- updated Authors.md to include some AML details
- added vowpalwabbit to AML skipped tests
- In skipped table, changed column title OS to Test Environment so that AML skips could use existing column

### Checklist:

- [x] I have followed the [contribution guidelines](../CONTRIBUTING.md) and code style for this project.
- [x] I have added tests covering my contributions.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,5 +89,19 @@ This project welcomes contributions and suggestions. Before contributing, please
 | **Windows GPU** | master | [![Status](https://msdata.visualstudio.com/AlgorithmsAndDataScience/_apis/build/status/nightly_gpu_win?branchName=master)](https://msdata.visualstudio.com/AlgorithmsAndDataScience/_build/latest?definitionId=6756) | | staging | [![Status](https://msdata.visualstudio.com/AlgorithmsAndDataScience/_apis/build/status/nightly_gpu_staging_win?branchName=staging)](https://msdata.visualstudio.com/AlgorithmsAndDataScience/_build/latest?definitionId=6761) |
 | **Windows Spark** | master | [![Status](https://msdata.visualstudio.com/AlgorithmsAndDataScience/_apis/build/status/nightly_spark_win?branchName=master)](https://msdata.visualstudio.com/AlgorithmsAndDataScience/_build/latest?definitionId=6757) | | staging | [![Status](https://msdata.visualstudio.com/AlgorithmsAndDataScience/_apis/build/status/nightly_spark_staging_win?branchName=staging)](https://msdata.visualstudio.com/AlgorithmsAndDataScience/_build/latest?definitionId=6754) |
 
+## AzureML Build Status
```
Comment (gramhagen) [inline README.md]: I think we should replace the old ones with the new ones, also prefer to remove the bp- prefix, what i'd like to see is just one integrated table with the unit test status. in the table it's not critical to differentiate where the test is run. the surrounding documentation is fine to spell that out.

File: README.md [hunk 2]
```diff
@@ -90,5 +89,15 @@ This project welcomes contributions and suggestions. Before contributing, please
...
```
Comment (miguelgfierro) [inline README.md]: maybe we could add a line under build status to differentiate  how the standard VM are being done vs how the AzureML tests are being done

File: tests/README.md [hunk 3]
```diff
@@ -8,6 +8,12 @@ This project uses unit, smoke and integration tests with Python files and notebo
 
 For more information, see a [quick introduction to unit, smoke and integration tests](https://miguelgfierro.com/blog/2018/a-beginners-guide-to-python-testing/). To manually execute the unit tests in the different environments, first **make sure you are in the correct environment as described in the [SETUP.md](../SETUP.md)**. 
 
+AzureML is also used to run the existing unit, smoke and integration tests as-is. AzureML benefits include managing the compute environment by automatically turning it on/off, scaling, automatic logging of artifacts from test runs and more. Azure DevOps is used as a control plane to provide information to scripts to configure and run the tests as-is on AzureML.  A separate set of pipelines was created to run the tests on AzureML and parameters to configure AzureML are defined in the pipeline yml files. There are two scripts used with each pipeline:
```
Comment (miguelgfierro) [inline tests/README.md]: how can we make the AzureML tests more explicit? Do we have any metrics of test time on a normal VM vs AzureML? In the past we were thinking of doing a blog post about this. Maybe we can have a section explaining the AzureML tests and adding the comments there.

**Comment by miguelgfierro** [comment]
@heatherbshapiro can you take a look at the writing to make sure we are promoting AzureML in the correct way?

**Pull request merged**